.github/workflows/nightly.yml

```diff
@@ -0,0 +1,60 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: '0 23 * * *'
```

Comment:
For debugging purpose, you can enable it temporarily for PR trigger (to see if it compiles properly)

Reply:
Sure